lambdas/tests/unit/enums/test_edge_presign_values.py

```diff
@@ -1,16 +1,30 @@
-# test_enums.py
-
 from enums.lambda_error import LambdaError
 
 ENV = "test"
+EXPECTED_ENVIRONMENT = ENV
```

Comment:
Seems like a number of these defined vars are not being used. The imports from the test handler are :

 EXPECTED_DOMAIN,
 EXPECTED_EDGE_MALFORMED_ERROR_CODE,
 EXPECTED_EDGE_MALFORMED_ERROR_MESSAGE,
 EXPECTED_EDGE_MALFORMED_HEADER_ERROR_CODE,
 EXPECTED_EDGE_MALFORMED_HEADER_ERROR_MESSAGE,
 EXPECTED_EDGE_MALFORMED_QUERY_ERROR_CODE,
 EXPECTED_EDGE_MALFORMED_QUERY_ERROR_MESSAGE,
 EXPECTED_EDGE_NO_ORIGIN_ERROR_CODE,
 EXPECTED_EDGE_NO_ORIGIN_ERROR_MESSAGE,
 TABLE_NAME,
 VALID_EVENT_MODEL,